resources/bin/crafter.sh

```
@@ -670,46 +657,27 @@ function doRestore() {
-
-      # Stop DB
-      banner "Stopping DB"
-      kill $(cat mariadb4j.pid)
-      sleep 10
+      cecho "Users backup does not exists. Skipping restore users\n" "warning"
     fi
+    abortOnError
+
+    # Stop DB
+    banner "Stopping DB"
+    kill $(cat mariadb4j.pid)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)